### PR TITLE
Bug fix - ConstraintEditor not working in case of object or multiple …

### PR DESCRIPTION
**Pull request by NetzerRosenfeld**
…field like LockPoint. If you type both x field and y field, only one field will be passed.

### CODE REVIEW

**Review by gauravkdeshmukh — COMMENTED** [review]
It seems like this commit is creating a new object for values and updating it.
Also, Is values always accessible in this function? May be we need to pass values as a parameter in the function!
Should we should use Object.assign() for creating a new object ?

**Comment by NetzerRosenfeld** [comment]
@gauravkdeshmukh Hi, I'm not sure I understood how you want to implement it. I tried to add Object assign without success, Did you mean to use it not without the setValues  of react?

**Comment by gauravkdeshmukh** [comment]
Hi @NetzerRosenfeld, I think we can achieve bugfix; by doing just a typo-fix.
`setValues({...values, [name]: value});`
I think the earlier implementation was missing the 's' in 'values'. 
The spread operator should work similarly to Object.assign(); Please try this, if it doesn't work then we can try 
```
Object.assign(values, {...values, [name]: value}); 
setValues(values);
```

File: web/app/sketcher/components/ConstraintEditor.jsx [hunk 1]
```diff
@@ -30,7 +30,16 @@ export function ConstraintEditor() {
   const {viewer} = useContext(SketcherAppContext);
 
   const setValue = (name, value) => {
-    setValues({...value, [name]: value});
+    //setValues({...value, [name]: value});Bug! not working in case of object or multiple field like LockPoint.
```
Comment (gauravkdeshmukh) [inline web/app/sketcher/components/ConstraintEditor.jsx:33]:
```suggestion
    setValues({...values, [name]: value});
```

Comment (NetzerRosenfeld) [reply web/app/sketcher/components/ConstraintEditor.jsx]: @gauravkdeshmukh Looks great, I change it.

**Review by gauravkdeshmukh — APPROVED** [review]

**Comment by gauravkdeshmukh** [comment]
@xibyte Can you review these changes? Thank you 🙂

**Comment by xibyte** [comment]
@gauravkdeshmukh @NetzerRosenfeld - thanks for the PR.

**Review by xibyte — APPROVED** [review]

**Pull request merged**